deploy-manage/remote-clusters/eck-remote-clusters-to-external.md

```diff
@@ -271,31 +230,12 @@ If you intend to use `sniff` mode, configure it through the [{{es}} API](#using-
 
 To add a remote cluster, use the [cluster update settings API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-cluster-put-settings). Configure the following fields:
 
-* **Remote cluster alias**: The cluster alias must match the one you configured when [adding the API key in the local cluster's keystore](#configure-local-cluster).
+* **Remote cluster alias**: The cluster alias must match the `<remote-cluster-name>` you configured when [adding the API key in the local cluster's keystore](#configure-local-cluster).
 * **mode**: Use `proxy` mode in almost all cases. `sniff` mode is only applicable when the remote cluster is self-managed and the local cluster can reach the nodes’ publish addresses directly.
 * **proxy_address**: Identify the endpoint of the remote cluster, including the hostname, FQDN, or IP address, and the port. Both IPv4 and IPv6 addresses are supported.
 
-  :::::::{applies-switch}
-
-  ::::::{applies-item} ess:
-  Obtain the endpoint from the **Security** page of the ECH deployment you want to use as a remote. Copy the **Proxy address** from the **Remote cluster parameters** section, and replace its port with `9443`, which is the port used by the remote cluster server interface.
-
-  ::::::
-
-  ::::::{applies-item} ece:
-  Obtain the endpoint from the **Security** page of the ECE deployment you want to use as a remote. Copy the **Proxy address** from the **Remote cluster parameters**, and replace its port with `9443`, which is the port used by the remote cluster server interface.
-  ::::::
-
-  ::::::{applies-item} eck:
-  Use the FQDN or IP address of the LoadBalancer service, or similar resource, you created to [expose the remote cluster server interface](#enable-rcs).  
-  ::::::
-
-  ::::::{applies-item} self:
-  The endpoint depends on your network architecture and the selected connection mode (`sniff` or `proxy`). It can be one or more {{es}} nodes, or a TCP (layer 4) load balancer or reverse proxy in front of the cluster, as long as the local cluster can reach them over port `9443`.
-
-  If you are configuring `sniff` mode, set the **seeds** parameter instead of **proxy_address**. Refer to the [connection modes](./connection-modes.md) documentation for details and connectivity requirements of each mode.
-  ::::::
-  :::::::
+  :::{include} _snippets/eck_rcs_external_endpoint_switch.md
```

Comment:
something funky is happening with this switch. I think it's because you're using it twice on the same page - it doesn't like that because the ID is not unique. you can try playing around with it a little but I think it might be a bug.

(when I click on the tabs, it's hopping up to the `kibana` section every time, and the tab body isn't rendering)

<img width="740" height="139" alt="image" src="https://github.com/user-attachments/assets/a1213d18-9b73-4cac-890d-4c1ba31b7160" />

Reply:
I reported it on https://github.com/elastic/docs-builder/issues/2334 and it's already fixed! Hopefully the fix will be live soon :)